Calibration/HcalCalibAlgos/test/isoTrackRecoAnalysis_cfg.py

```diff
@@ -0,0 +1,86 @@
+import FWCore.ParameterSet.Config as cms
+process = cms.Process('ANALYSIS')
+
+process.load('Configuration.StandardSequences.Services_cff')
+process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
+
+process.MessageLogger = cms.Service("MessageLogger",
+    destinations = cms.untracked.vstring('warnings','errors',
+                                         'cout','cerr'),
+    categories = cms.untracked.vstring('HcalIsoTrack'), 
+    debugModules = cms.untracked.vstring('*'),
+    warnings = cms.untracked.PSet(
+        placeholder = cms.untracked.bool(True)
+    ),
+    default = cms.untracked.PSet(
+
+    ),
+    errors = cms.untracked.PSet(
+        placeholder = cms.untracked.bool(True)
+    ),
+    cerr = cms.untracked.PSet(
+        optionalPSet = cms.untracked.bool(True),
+        INFO = cms.untracked.PSet(
+            limit = cms.untracked.int32(0)
+        ),
+        noTimeStamps = cms.untracked.bool(False),
+        FwkReport = cms.untracked.PSet(
+            optionalPSet = cms.untracked.bool(True),
+            reportEvery = cms.untracked.int32(500),
+            limit = cms.untracked.int32(10000000)
+        ),
+        default = cms.untracked.PSet(
+            limit = cms.untracked.int32(10000000)
+        ),
+        Root_NoDictionary = cms.untracked.PSet(
+            optionalPSet = cms.untracked.bool(True),
+            limit = cms.untracked.int32(0)
+        ),
+        FwkJob = cms.untracked.PSet(
+            optionalPSet = cms.untracked.bool(True),
+            limit = cms.untracked.int32(0)
+        ),
+        FwkSummary = cms.untracked.PSet(
+            optionalPSet = cms.untracked.bool(True),
+            reportEvery = cms.untracked.int32(1),
+            limit = cms.untracked.int32(10000000)
+        ),
+        threshold = cms.untracked.string('INFO')
+     ),
+     cout = cms.untracked.PSet(
+        threshold = cms.untracked.string('INFO'),
+        noTimeStamps = cms.untracked.bool(True),
+        INFO = cms.untracked.PSet(
+            limit = cms.untracked.int32(0)
+        ),
+        DEBUG = cms.untracked.PSet(
+            limit = cms.untracked.int32(0)
+        ),
+        HcalIsoTrack = cms.untracked.PSet(
+            limit = cms.untracked.int32(-1)
+       )
+    )
+)
+
+process.load("Configuration.Geometry.GeometryIdeal_cff")
+process.load('Configuration/StandardSequences/GeometryExtended_cff')
+process.load('Configuration/StandardSequences/MagneticField_38T_cff')
+process.GlobalTag.globaltag = 'START53_V15::All'
```

Comment:
I am a bit confused with all the comments. Please let me know if the following set is OK in the input python file.
process.load("Configuration.Geometry.GeometryIdeal_cff")
process.load("Configuration.StandardSequences.MagneticField_cff")
process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
from Configuration.AlCa.autoCond_condDBv2 import autoCond
process.GlobalTag.globaltag=autoCond['run2_mc']

If not what could be. We have estimated the event size and it is not changed much. We have been copying some stuff and storing them - now we do not copy those stuff and store them directly. We can report on the data volume etc in the next AlCa meeting if you want.

Sunanda
________________________________
From: Marco Musich [notifications@github.com]
Sent: 07 May 2015 12:37
To: cms-sw/cmssw
Cc: Sunanda Banerjee
Subject: Re: [cmssw] bsunanda:Run2-alca11 Tune+test several AlCaReco jobs and prepare DQM plots for IsoTrack Trigger (#8943)


In Calibration/HcalCalibAlgos/test/isoTrackRecoAnalysis_cfg.py<https://github.com/cms-sw/cmssw/pull/8943#discussion_r29840921>:

> +        INFO = cms.untracked.PSet(
> +            limit = cms.untracked.int32(0)
> +        ),
> +        DEBUG = cms.untracked.PSet(
> +            limit = cms.untracked.int32(0)
> +        ),
> +        HcalIsoTrack = cms.untracked.PSet(
> +            limit = cms.untracked.int32(-1)
> +       )
> +    )
> +)
> +
> +process.load("Configuration.Geometry.GeometryIdeal_cff")
> +process.load('Configuration/StandardSequences/GeometryExtended_cff')
> +process.load('Configuration/StandardSequences/MagneticField_38T_cff')
> +process.GlobalTag.globaltag = 'START53_V15::All'


this is still pointing to v1

—
Reply to this email directly or view it on GitHub<https://github.com/cms-sw/cmssw/pull/8943/files#r29840921>.